lib/plugins/aws/invokeLocal/index.js

```diff
@@ -107,8 +109,26 @@ class AwsInvokeLocal {
         this.options.data);
     }
 
+    if (runtime === 'python2.7') {
+      return this.invokeLocalPython(
+        handlerPath,
+        handlerName,
+        this.options.data);
+    }
+
     throw new this.serverless.classes
-      .Error('You can only invoke Node.js functions locally.');
+      .Error('You can only invoke Node.js & Python functions locally.');
+  }
+
+  invokeLocalPython(handlerPath, handlerName, event) {
+    return new BbPromise(resolve => {
+      const python = spawn(path.join(__dirname, 'invoke.py'), [handlerPath, handlerName]);
```

Comment:
Good catch. I've updated to add this: https://github.com/serverless/serverless/pull/2937/commits/dd0ff02624eb8159ade3b633b72017a2c88a7248#diff-f41c172f712b450ef245164b90bdb5d5R126